roles/cdi/defaults/main.yml

```diff
@@ -1,3 +1,4 @@
+---
 cdi_namespace: "{{ cdi_image_namespace | default('kube-system') }}"
```

Comment:
should be "cdi" instead of "kube-system"

Reply:
done